### DISCUSSION TITLE
The rate limit of primary domain javlibrary.com can be solved by accessing temporary domains

### DISCUSSION THREAD

**Discussion by ramondsq** [hobbyfang/javOldDriver#64] [General]
As discovered not long ago, the rate limit of the primary domain **javlibrary.com** is too high to influence the normal browsing. 
And I found out that it can be solved by accessing the temporary domains published in [zlibz.com](https://zlibz.com/) instead. 
The letters on the right side of the page combined with `.com` is the temporary domain.

**Comment by hobbyfang** [comment]
临时域名也不能解决列表问题，只是比主域名相对好一点点。
